docs/tutorials/getting-started.md

```diff
@@ -0,0 +1,248 @@
+```
+
+### Installing Morpho
+
+To install the latest Morpho version, run the command
+```
+$ nix profile install github:input-output-hk/ECIP-Checkpointing
+```
+
+Then verify that the installation worked with
+```
+$ morpho-checkpoint-node --help
+```
+
+### Installing Mantis
+
+
+Morpho was developed in conjunction with [Mantis](https://mantisclient.io/), which implements the server side of the [RPC interface](../references/rpc.md) required by Morpho. To install a Mantis version which is compatible with the Morpho version, run the command
+
+```
+$ nix profile install github:input-output-hk/ECIP-Checkpointing#mantis
+```
+
+Then verify that the installation worked with
+```
+$ mantis cli --help
+```
+
+## Running a single-node Morpho federation (without Mantis)
+
+Morpho is intended to be run by a number of nodes, but we will start with just one node and add further ones later. Nodes are 0-indexed, so our one and only node will have id 0. Let's create a new directory for our node, which we will use in the following sections:
+```
+$ mkdir ~/morpho/node-0
+$ cd ~/morpho/node-0
+```
+
+### Generating a key pair
+
+Any Morpho node needs a private/public key pair for [signing votes](../explanations/crypto.md#vote-signing). Currently Morpho doesn't have its own way to generate such a key pair, but we can use Mantis for that instead, which uses exactly the same signatures scheme.
+
+To generate a key pair with mantis, run the following command:
+```
+$ mantis cli generate-key-pairs
+06334908ff142cd6ce98127874f61f178eaddfbb952d03dd7b4df61e9aaef908
+813abe586ed99372d5642408c0b883a177e2a5c4fd95ed9f210226a5265e5a5a26f230bf52473105da493d08ce1db60da07bc7e96ed52461fc0857ccabb4f0ac
+```
+
+This gives us a fresh private key on the first line, along with its public key on the second line, which we will write to files for later:
+
+```
+$ echo 06334908ff142cd6ce98127874f61f178eaddfbb952d03dd7b4df61e9aaef908 \
+  > private-key
+$ echo 813abe586ed99372d5642408c0b883a177e2a5c4fd95ed9f210226a5265e5a5a26f230bf52473105da493d08ce1db60da07bc7e96ed52461fc0857ccabb4f0ac \
+  > public-key
+```
+
+### Creating a topology file
+
+Morpho has to know how to connect to other Morpho nodes, which is done with a [topology file](../references/topology.md). Since we only use a single node for the moment, our topology file will be very simple. Note that this file can be shared between all Morpho nodes. Let's create a `~/morpho/topology.json` file for our single-node topology with the following contents:
+```json
+[
+  {
+    "nodeId": 0,
+    "producers": []
+  }
+]
+```
+
+### Creating the config file
+
+In this section we will create the Morpho configuration file. All configuration fields are explained in detail [here](../references/configuration.md). The first thing we need to do to start a new Morpho network is to decide on a set of parameters which can't be changed later. The most important ones are:
```

Comment:
detail [here] -> [the configuration page]
which -> that

Reply:
I'm again a bit confused here, when should it be `which` vs `that`?